pages.zh/common/mvn.md

```diff
@@ -14,7 +14,7 @@
 
 - 编译和打包，跳过单元测试：
 
-`mvn package -Dmaven.test.skip=true`
+`mvn package -skipTests`
```

Comment:
missing `-D`

Reply:
Thank you. 👍
Fixed in c67a66063c566208edff7e58f4bb8c715c080023.